style.css

```diff
@@ -42,4 +60,14 @@ h3 {
 
 #colorPalette {
   line-height: 30px;
+}*/
```

Comment:
There are some redundant rules here in your CSS that you can DRY out. Try using class selectors rather than IDs, so you can reuse them in many places.